src/tests/ftest/security/pool_security_groups.yaml

```diff
@@ -5,24 +5,15 @@ hosts:
     - server-A
   test_clients:
     - server-B
-  transport_config:
-    - allow_insecure: False
-    - ca_cert: ~/.daos/daosCA.crt
-    - cert: ~/.daos/daos_agent.crt
-    - key: ~/.daos/daos_agent.key
-    - server_name: server
-timeout: 600
+timeout: 1200
 server_config:
   name: daos_server
   port: 10001
-daos_server:
-  start:
-     insecure: False
 pool_acl:
-  scm_size: 134217728
+  scm_size: 1334217728
   user_prefix: daos_ci
-  num_user: 1
-  num_group: 1
+  num_user: 5
+  num_group: 5
```

Comment:
Is this change needed in order to enable secure mode?

Reply:
no

Reply:
So it needs to be fixed before the conversation can be resolved -- by the person who started the conversation, once they have confirmed that it as been fixed.